industry_restaurant/demo/pos_order_line.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="pos_order_line_18" model="pos.order.line">
+        <field name="order_id" ref="pos_order_6"/>
+        <field name="product_id" ref="product_product_47"/>
+        <field name="full_product_name">Combo(Chicken Burger + Coke)</field>
+        <field name="qty">2.0</field>
+        <field name="price_unit">2.0</field>
+        <field name="price_subtotal">4.0</field>
+        <field name="price_subtotal_incl">4.60</field>
+    </record>
+    <record id="pos_order_line_25" model="pos.order.line">
+        <field name="order_id" ref="pos_order_8"/>
+        <field name="product_id" ref="product_product_47"/>
+        <field name="full_product_name">Combo(Chicken Burger + Coke)</field>
+        <field name="qty">2.0</field>
+        <field name="price_unit">2.0</field>
+        <field name="price_subtotal">4.0</field>
+        <field name="price_subtotal_incl">4.60</field>
+        <field name="is_total_cost_computed" eval="True"/>
```

Comment:
Seems to be computed from `_compute_total_cost`, right?

Reply:
yaa

Reply:
it is not getting computed